apps/src/dance/lottie/LottieDancerUtils.ts

```diff
@@ -268,38 +308,39 @@ export function applyColorMapping(
   );
 }
 
-/** Loads head.png as a data URL and detects natural size (falls back to 1024×1024). */
-export async function fetchHeadImageInfo(
-  headUrl: string | null
-): Promise<HeadImageInfo | null> {
-  if (!headUrl) {
+/** Loads png as a data URL and detects natural size (falls back to 1024×1024). */
+export async function fetchDataUrl(url?: string): Promise<string | null> {
+  if (!url) {
     return null;
   }
   try {
-    const response = await fetch(headUrl);
+    const response = await fetch(url);
     if (!response.ok) {
       return null;
     }
 
     const blob = await response.blob();
-    const dataUrl = await new Promise<string>(resolve => {
+    return await new Promise<string>(resolve => {
       const fileReader = new FileReader();
       fileReader.onload = () => resolve(fileReader.result as string);
       fileReader.readAsDataURL(blob);
     });
-
-    // We’ve standardized head asset dimensions, so no image probe needed.
-    return {dataUrl, width: DEFAULT_HEAD_W, height: DEFAULT_HEAD_H};
```

Comment:
These dimensions weren't actually related to the loaded image. We were really just get the data URL and passing some constants, so I've renamed and cleaned up.